torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -34,6 +36,18 @@ struct NoGilSection {
   PyThreadState* state = nullptr;
 };
 
+struct MaybeDevice {
```

Comment:
Why can't we just use `c10::optional<c10::Device>` instead of this?

Reply:
SG